dev/io.openliberty.microprofile.openapi.2.0.internal.services/src/io/openliberty/microprofile/openapi20/internal/services/impl/OpenAPIInfoConfig20Impl.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.openapi20.internal.services.impl;
+
+import org.osgi.service.component.annotations.Component;
+
+import io.openliberty.microprofile.openapi20.internal.OpenAPIInfoConfigImpl;
+import io.openliberty.microprofile.openapi20.internal.services.OpenAPIInfoConfig;
+
+@Component(configurationPid = "io.openliberty.microprofile.openapi", service = OpenAPIInfoConfig.class)
```

Comment:
More a question than a comment, I notice you like to have one component for each part of OpenAPI config, rather than a centralized one. 

Does having more multiple OSGi components have any performance implications?

Reply:
It might do, I'll raise an issue to consolidate them.